.editorconfig

```diff
@@ -0,0 +1,36 @@
+# https://editorconfig.org/
+# specification: https://editorconfig-specification.readthedocs.io/en/latest/
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{c,h,hh,cc,cci}]
+indent_style = space
+indent_size = 4
+
+[Makefile]
+indent_style = tab
+
+[*.am]
+indent_style = tab
+
+[configure.ac]
+indent_style = space
+indent_size = 2
+
+[*.awk]
+indent_style = space
+indent_size = 4
+
+[*.{pl,pl.in,pm}]
+indent_style = space
+indent_size = 4
+
+[*.m4]
+indent_style = space
+indent_size = 4
```

Comment:
I suggest merging all these identical rules with the C/C++ block above. This would send a pretty clear signal that this is the formatting we prefer "by default" (i.e. unless there are good reasons to use something else).

I do not insist on this change.

Reply:
I see your point. At the same time, it would be a pretty long, "kitchen sink" style of extensions and languages; I find separating them by language to more readable.

OTOH, there is a bug in the *.m4 line, which can furthermore be merged with *.ac. I'll do that.